src/trivia/libraries/trivia/trivia.go

```diff
@@ -0,0 +1,91 @@
+package trivia
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+	"time"
+	"trivia/libraries/prize"
+	"trivia/libraries/problems"
+)
+
+const limitSeconds = 60
+const reward = 10
+const penalty = 5
+
+// Trivia game
+func Trivia() {
+	timeLimit := flag.Int("limit", limitSeconds, "the time limit for the quiz in seconds")
+	flag.Parse()
+
+	problems, err := problems.GetProblems()
+	if err != nil {
+		exit(err)
+	}
+
+	// ENTER to start
+	fmt.Printf("\n***************** 1980s TV TRIVIA QUIZ *****************\nYou win $%d for every correct answer!\nYou loose $%d for every wrong answer (no worries, you won't go negative, it's all about winning! :D)\nTime limit is %d seconds.\n\nREADY SET GO!(press ENTER to start)\n\n", reward, penalty, limitSeconds)
+	reader := bufio.NewReader(os.Stdin)
+	start, _ := reader.ReadString('\n')
+	timer := time.NewTimer(time.Duration(*timeLimit) * time.Second)
+	timer.Stop()
+
+	if start[0] == 10 {
+		timer.Reset(time.Duration(*timeLimit) * time.Second)
+	}
+	correct := 0
+	readQuestions := 0
+
+	prize := &prize.Prize{0}
+
+problemloop:
```

Comment:
labels like this are usually a bad code smell, they confuse people. I see why you did this here but there are some other options for this sort of control in go. You should try a version where you put most or all of this loop into a separate function and then use an early return from the function to get out. So the function is something like "doAllTheProblems" and when timer runs out it just returns